packages/constants/CategoryDescriptionConstants.ts

```diff
@@ -0,0 +1,42 @@
+export const CATEGORY_DESCRIPTION = {
```

Comment:
Does this mean the JS side will be reading from this constants file from now on?

Reply:
Yes

Reply:
Will this be a chore in the upcoming sprint? Or are you already working on it?

Reply:
This is part of the story.